routes/project/location/firewall.rb

```diff
@@ -31,8 +30,7 @@ class Clover
         firewall.destroy
 
         if api?
-          response.status = 204
-          nil
+          204
         else
           {message: "Deleting #{firewall.name}"}
         end
```

Comment:
nit: Since it's a response to an AJAX request, a 204 No Content status is also acceptable. So we can merge branches